multi-int-store/src/test/java/com/connexta/multiintstore/TestUtilConfig.java

```diff
@@ -9,38 +9,23 @@
 import static org.mockito.Mockito.mock;
 
 import org.apache.solr.client.solrj.SolrClient;
-import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Primary;
 import org.springframework.context.annotation.Profile;
-import org.springframework.web.client.RestTemplate;
 import software.amazon.awssdk.services.s3.S3Client;
 
 @Configuration
-@Profile("test")
```

Comment:
Kinda the same question here. Is there a benefit to just profile the beans instead of the config class? Even if they're the only methods in the whole class?

Reply:
See https://github.com/connexta/multi-int-store/pull/96/files/13d6c7db6a2fd6424f066ec17e9c05cbeda0d914#r302330941. Both itest classes need the mocked S3 client. Only one itest class need the mocked solr client.